alpenhorn/util.py

```diff
@@ -43,7 +43,11 @@ def run_command(cmd, **kwargs):
     stdout_val, stderr_val = proc.communicate()
     retval = proc.returncode
 
-    return retval, stdout_val, stderr_val
+    return (
+        retval,
+        stdout_val.decode(errors="replace"),
+        stderr_val.decode(errors="replace"),
+    )
```

Comment:
Why did this get added? Does this affect stdout for other commands written before James' code?